cscs-checks/libraries/math/dgemm.py

```diff
@@ -33,9 +33,26 @@ def __init__(self):
         }
 
         self.maintainers = ['AJ', 'VH', 'VK']
-        self.tags = {'diagnostic'}
+        self.tags = {'benchmark', 'diagnostic'}
 
     def setup(self, partition, environ, **job_opts):
+
+        if environ.name.startswith('PrgEnv-gnu'):
+            envname = 'PrgEnv-gnu'
```

Comment:
As far as I can see, the `envname` variable is defined but never used.

Reply:
Is it used if I select on the command line `-p PrgEnv-gnu` when I invoke `reframe`?